### ISSUE TITLE
Dimension crossing issues

### ISSUE THREAD

**Issue by GuntherDW** [jadedcat/MF3Configs#21]
Hi

There seems to be a weird issue with some mod in this pack.
We're running a MF3 1.1.35 server, and it seems like some mod is screwing with the world pointers for a player.

For example, I have 4 beacons on a blood altar, all 4 of them are active.
If I then go through the nether portal, or to any particular dimension, either with blood magic's teleposers or using the mods' own way to get there, it seems like a couple of issues occur.
Sometimes I will only get a select few chunks up to something like the viewdistance. I have to relog to get the other chunks to load.
The beacon effects from said blood altar also seem to effect me whether or not this chunk loading bug occurs, I do have to travel to the exact coordinates those beacons are in the other dimension though. I can provide a screenshot, but it won't really be of any use other than to show that I get the effects.
It also sometimes will not allow me to break blocks in that dimension, and mobs will spawn/despawn randomly. I can still kill them and collect the drops accordingly (and they can hurt/kill me as well).

There is a smeltery nearby those chunks that also seems to rip my face off whenever i get "inside" of it in the other dimensions. It correctly adds blood to the smeltery contents however.

The TPS on the server is fine, we get absolutely no "can't keep up" messages in the server as of yet.

The weird thing is that the blood altar's beacons and altar itself are not chunkloaded (and are out of spawn range), yet the effects still apply.

"Optimizing" mods like FastCraft and Optifine are not installed. The changes we did however do is include AE2, enable thaumcraft's "hard mode nodes" and fix the agricraft seeds' description in the config so that they don't error out and don't exist in the world. There are however no AE2 blocks in the blood altar chunk, nor are there any in the smeltery chunk.

EDIT: Using the precision sledgehammer I could see where my pen of cows is located in the other worlds, and they it seems the attacks from the cows also work cross-dimensionally. I must stress that after relogging it correctly places me in the dimension I'm in, but if I travel to another dimension from the overworld it is like my body is in 2 places at the same time.

**Comment by ghost** [comment]
This is either a client, or a server rendering issue. Are you sure you have enough RAM allocated to allow it to run smoothly?

Does doing f3 + a help (force reloading of chunks).

**Comment by ghost** [comment]
I don't know how, but you broke everything.

**Comment by ghost** [comment]
Okay, so hypothesis we've come up with

Tampering with the modpack by adding a new mod to a "pre-existing" world, probably caused forge to re-parse all the block and item ID's, possibly causing a memory leak inside the world data? Main suggestion being, don't tamper with the modpack (it was designed to NOT have AE2 for a reason), have a world reset (the ouchy kind).

Sadly, since the modpack has been tampered with, we cannot provide any support regarding this, but the above suggestions are worth a try.

**Comment by LemADEC** [comment]
You've 2 symptoms here:
1- chunks not loading on client side: it's a known issues with certain server modifications, notably the chicken chunks mod.
2- worlds are 'overlapping'. When player changes dimension, minecraft reload the player from scratch. Some mods (notably AM2), plugins (many) and server (Cauldron) are known to improperly handle that and cause such weird issues.
Either way, issue isn't from the mopack. You need to remove plugins/mods to narrow down the issue and report to the proper mod/plugin/server support.

**Comment by GuntherDW** [comment]
The client had 4GB of RAM allocated, the server around 8GB, I do think that's enough.
I'm also using the standard server setup, nothing like Cauldron or the likes.

The reason AE2 wasn't in the pack wasn't because of technical reasons though, but because it didn't fit in with the playstyle. The pack isn't geared towards tech.

ChickenChunks wasn't installed, the only chunkloading mod I used was the MFR chunkloader blocks.

But the map is pretty much "dead" by now, seeing as this was a pretty semi-private server only I played on it really towards the end. I do have a backup of everything if you want.

EDIT: Ctrl+a (reloading of chunks) doesn't do anything. Never did really. I tried to be as detailed as I could in the things I tried in the original report.
